tests/integration/targets/aws_glue_job/tasks/main.yml

```diff
@@ -0,0 +1,402 @@
+---
+- name: run aws_glue_job integration tests
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+      purge_tags: no
```

Comment:
```suggestion
```

Adding purge_tags here implies that the default behaviour isn't what people might expect (and in this case means you won't notice that you're defaulting to nuking tags.)